src/i18n/en-US/grbReview.ts

```diff
@@ -258,7 +260,24 @@ export default {
         ]
       }
     },
-    completeAndBeginReview: 'Complete and begin review'
+    completeAndBeginReview: 'Complete and begin review',
+    step4: {
```

Comment:
Not a fan of naming this `step4`.  I think we should stick to the convention of naming it after the component it references